### PR TITLE
[MGTL] Fix GeoMapper tri/line intersection memleak

### PR DESCRIPTION
**Pull request by endJunction**
Fix a memory leak detected by clang's address sanitizer in the testrunner.

### CODE REVIEW

File: MeshGeoToolsLib/GeoMapper.cpp [hunk 1]
```diff
@@ -201,7 +201,7 @@ double GeoMapper::getMeshElevation(
 	    _grid->getNearestPoint(MathLib::Point3d{{{x, y, 0}}});
 	const std::vector<MeshLib::Element*> elements(
 	    _surface_mesh->getNode(pnt->getID())->getElements());
-	GeoLib::Point* intersection(nullptr);
+	std::unique_ptr<GeoLib::Point> intersection;
```
Comment (rinkk) [inline MeshGeoToolsLib/GeoMapper.cpp]: is this automatically a nullptr? (otherwise this will fail a few lines below)

Comment (endJunction) [inline MeshGeoToolsLib/GeoMapper.cpp]: Yes. The default ctor constructs a std::unique_ptr that owns nothing. [cppref](http://en.cppreference.com/w/cpp/memory/unique_ptr/unique_ptr)

**Comment by chleh** [comment]
:fast_forward: although it's not good that you mixed refactorings and other changes in a single commit. ;-)

**Comment by endJunction** [comment]
@chleh I must object. Only unique_ptr is changed.

**Comment by chleh** [comment]
@endJunction: I must admit, you are right. And I must realize that Github's color-diff sucks! ^^

**Comment by chleh** [comment]
Jenkins, test this please.

**Pull request merged**